en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md

```diff
@@ -1183,7 +1183,20 @@ You have to run the following migration client to update the API Manager artifac
 
 3. Prior to API-M migration, run the below command to execute the pre-migration step that will validate your old data.
 
-    - Available validators: `apiAvailabilityValidation`, `apiDefinitionValidation`
+    ??? info "Pre-validators"
+        
+        | API Validators                            |                                          |                                      |
+        | ----------------------------------------- | ---------------------------------------- | ------------------------------------ |
+        | **CLI Tag**                               | **Pre-validator**                        | **Purpose**                          |
+        | `apiDefinitionValidation`                 | API Definition Validator                 | Validates the API definitions are up to standards so that issues are not encountered during migration. Validations are done to check if APIs  have valid OpenAPI, WSDL, Streaming API or GraphQL API definitions. |
```

Comment:
```suggestion
        | `apiDefinitionValidation`                 | API Definition Validator                 | Validates if the API definitions are up to standards so that issues are not encountered during migration. Validations are done to check if APIs have valid OpenAPI, WSDL, Streaming API, or GraphQL API definitions. |
```